translate/storage/php.py

```diff
@@ -285,6 +285,8 @@ def getoutput(self, indent="", name=None):
             fmt = "{1}{2}{1},\n"
         elif "->" in self.name:
             fmt = "{0} => {1}{2}{1},\n"
+            if name is not None:
+                name = name[0] + phpencode(name[1:-1]) + name[-1]
```

Comment:
It should include escape type:

```suggestion
                name = name[0] + phpencode(name[1:-1], name[0]) + name[-1]
```

Can you please also add a test case for double-quoted string?